grpc/v1/webserver.proto

```diff
@@ -68,10 +68,13 @@ message SearchOptions {
   // be sent and then the behaviour will revert to the normal streaming.
   google.protobuf.Duration flush_wall_time = 7;
 
-  // Trim the number of results after collating and sorting the
-  // results
+  // Truncates the number of documents (i.e. files) after collating and
+  // sorting the results.
   int64 max_doc_display_count = 8;
 
+  // Truncates the number of matchs after collating and sorting the results.
+  int64 max_match_display_count = 16;
```

Comment:
@isker I wouldn't mind using a number from the lower range. @keegancsmith WDYT? AFAIK the convention is to use field numbers 16 or higher for fields that might change or are used less frequently because these numbers are less efficiently encoded. I imagine this to be a frequently used field by many clients.

Reply:
1-15 are taken.

Reply:
🤦 never mind. I was obviously too lazy to expand the diff ;-)